src/buildApplication/buildApplication.ts

```diff
@@ -125,18 +125,18 @@ function printInvalidFunctionsErrorMessage(invalidFunctions: string[]): void {
 				.join('\n')}
 
 			If this is a Next.js project:
-
-			- you can read more about configuring Edge API Routes here: https://nextjs.org/docs/api-routes/edge-api-route
 			
-			- you can try enabling the Edge Runtime for a specific page by exporting the following from your page:
+			- You can try enabling the Edge Runtime for a specific route by exporting the following from your page:
 
-					export const config = { runtime: 'edge' };
+				export const runtime = 'edge';
```

Comment:
I didn't add the next.js version part because i just wanted to add a quick change so that it reflects the changes to the documentation. i guess i can remove these parts of the changes and leave it for another pr

Reply:
yes please so that we can make this a docs update only :) (also the error message is already broken anyways 😓)